frontend/src/components/game/Editor.tsx

```diff
@@ -63,6 +80,18 @@ function ResizableMonacoEditor(props: EditorProps) {
     });
   };
 
+  const handleEditorWillMount = (editor: any) => {
```

Comment:
nit: Is the editor any specific type? If so, add that type here - no problem if this isn't straightforward to find

Reply:
Did a little digging and was able to replace all the anys in this class - the types are pretty weird tho, like `typeof monaco` or `monaco.editor.IStandaloneCodeEditor`, so lmk what your thoughts are about that

Reply:
I think that's good - thanks for doing that digging!